components/enedis/actions/get-address/get-address.mjs

```diff
@@ -0,0 +1,23 @@
+import enedis from "../../enedis.app.mjs";
+
+export default {
+  type: "action",
+  key: "enedis-get-address",
+  version: "0.0.1",
+  name: "Get address",
+  description: "Returns the address of a client.",
+  props: {
+    enedis,
+    usage_point_id: {
+      propDefinition: [
+        enedis,
+        "usage_point_id",
+      ],
+    },
+  },
+  async run({ $ }) {
+    return await this.enedis.address(
```

Comment:
`return await` is something you should almost never use - [here's a good article explaining why](https://arthur.place/the-cost-of-return-await)
```suggestion
    return this.enedis.address(
```